### DISCUSSION TITLE
Zigbee Bulb Not Joining

### DISCUSSION THREAD

**Discussion by Haverer** [Koenkk/zigbee2mqtt#23348] [Devices]
Hi All 
Bought a Zigbee 3 bulb off Ali Express... compatible with Hue, Smart things etc. It automatically colour cycles and enters pairing mode, but won't join Zigbee2MQTT coordinator.... anything I can try?

**Comment by Haverer** [comment]
this is the bulb
eWelink Zigbee Smart Dimmable Bulb 85

**Comment by lux73** [comment]
any Zigbee Device should join the Network if following the zigbee Specs - even it's not supported and then not usable
